<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QWidget" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginDlg</string>
  </property>
  <widget class="QFrame" name="frame_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: rgb(255, 255, 255);
}


#frame_background{
	border-image: url(:/imageSource/back.jpg);
	border-radius:30px;
}
#frame_transparent{
	background-color: rgba(140, 140, 140, 140);
	border-radius:30px;
}
#frame_login{
	background-color: rgba(0, 255, 255, 80);
	border-radius:30px;
}
#label_login{
	background:transparent;
	color: rgb(255, 255, 255);
	font:30px;
}
QLineEdit{
	border:none;
	background:transparent;
	border-bottom:1px solid rgba(255, 255, 255, 200);
	color: rgb(255, 255, 255);
	font:20px;
}
QLineEdit:focus{
	border:none;
	background:transparent;
	border-bottom:1px solid rgba(255, 255, 255, 200);
	color: rgb(0, 0, 0);
	font:20px;
}
#but_login{
	color: rgb(255, 255, 255);
	border-radius:10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(170, 170, 255, 200), stop:1 rgba(255, 255, 255, 200));
}
#but_login:pressed{
	color: rgb(255, 255, 255);
	border-radius:10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(170, 170, 255, 200), stop:1 rgba(255, 255, 255, 200));
	padding-top:5px;
	padding-left:5px;
}
QToolButton{
	background:transparent;
	border:none;
}
QToolButton:hover{
	background:transparent;
	border:none;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 0, 0, 200), stop:1 rgba(255, 255, 255, 255));

}
#label_tip{
	background:transparent;
	color: rgb(255, 0, 0);
}
#tbut_show{
	background:transparent;
}
#tbut_show::indicator{
	width:40px;
	height:40px;
}

#tbut_show::indicator:unchecked:hover{
	border-image: url(:/imageSource/hide_default.png);
	background:transparent;
}
#tbut_show::indicator:unchecked{
	border-image: url(:/imageSource/hide_hover.png);
	background:transparent;
}

#tbut_show::indicator:checked{
	border-image: url(:/imageSource/show_default.png);
	background:transparent;
}
#tbut_show::indicator:checked:hover{
	border-image: url(:/imageSource/show_hover.png);
	background:transparent;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_transparent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>500</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_login">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>641</width>
       <height>401</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_login">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_admin">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>251</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>admin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>240</y>
        <width>251</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="but_login">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>320</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_tip">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>370</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="tbut_show">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>250</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>1066577840</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>1276926781</string>
      </property>
     </widget>
     <widget class="QPushButton" name="but_register">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>330</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#but_register{
	background:transparent;
}
#but_register:hover{
	background:transparent;
	color: rgb(85, 0, 255);
}
#but_register:pressed{
	background:transparent;
	color: rgb(85, 0, 255);
	padding-top:5px;
	padding-left:5px;
}
</string>
      </property>
      <property name="text">
       <string>注册账号</string>
      </property>
     </widget>
    </widget>
    <widget class="QToolButton" name="tbut_close">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>×</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbut_mini">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>0</y>
       <width>48</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>·</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
